### PR TITLE
Regenerate all apps with JHipster Native blueprint

### PR DESCRIPTION
**Pull request by mraible**
Related to https://github.com/jhipster/generator-jhipster/issues/16498#issuecomment-1030872667.

### CODE REVIEW

File: angular-webflux/angular.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
   "cli": {
     "cache": {
       "enabled": true,
-      "path": "./target/",
+      "path": "./target//angular/",
```
Comment (mraible) [inline angular-webflux/angular.json]: @mshima I'm not sure why this changed.

File: angular-webflux/package.json [hunk 2]
```diff
@@ -48,16 +48,16 @@
...
     "java:jar": "./mvnw -ntp verify -DskipTests --batch-mode",
-    "java:jar:dev": "npm run java:jar -- -Pdev,webapp",
-    "java:jar:prod": "npm run java:jar -- -Pprod",
+    "java:jar:dev": "npm run java:jar -- -Pnative,dev,webapp",
+    "java:jar:prod": "npm run java:jar -- -Pnative,prod",
     "java:war": "./mvnw -ntp verify -DskipTests --batch-mode -Pwar",
-    "java:war:dev": "npm run java:war -- -Pdev,webapp",
-    "java:war:prod": "npm run java:war -- -Pprod",
+    "java:war:dev": "npm run java:war -- -Pnative,dev,webapp",
+    "java:war:prod": "npm run java:war -- -Pnative,prod",
```
Comment (mraible) [inline angular-webflux/package.json]: @mshima A native app currently doesn't run if you build it with the `dev` profile, so I'm not sure we should add `native` to `dev` profiles.

File: angular-webflux/angular.json (hunk 1 above)
Comment (mshima) [reply angular-webflux/angular.json]: It’s a generator-jhipster change.
Angular was cleaning the entire target folder and builds were failing.

File: angular-webflux/package.json (hunk 2 above)
Comment (mshima) [reply angular-webflux/package.json]: Ok, let’s drop.

**Pull request merged**